### PR TITLE
Feature/as 1331 guidance page before

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-1243

## Description of change
- Create guidance page template
- Added content for 'Before Appeal' page
- Moved start of form to be guidance pages

Note: no components for previous/next pagination or page nav exist in Design system currently or in convient repo. Have not made component as these pages are only temporary until the service launches and pages end up on govuk site instead.

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: e2e-tests/cypress/integration/common/guidance-pages.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+When('the appellant navigates to the next page', () => {
+  cy.guidancePageNavigation('next');
+});
```
Comment (swdpcomputing) [inline e2e-tests/cypress/integration/common/guidance-pages.js:5]: Do we need to confirm what page has been navigated to?

File: e2e-tests/cypress/support/guidance-pages/guidancePageNavigation.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  let identifier;
+  switch (navigationType) {
+    case 'next':
+      identifier = '[data-cy="pagination-next"]';
+      break;
+    case 'previous':
+      identifier = '[data-cy="pagination-previous"]';
+      break;
+    case 'start':
+      identifier = '[data-cy="pagination-previous"]';
```
Comment (swdpcomputing) [inline e2e-tests/cypress/support/guidance-pages/guidancePageNavigation.js:11]: Is the data-cy supposed to be the same for lines 8 and 11?

**Review by rcmsoftwaredevelopment — DISMISSED** [review]

File: packages/forms-web-app/src/routes/index.js [hunk 3]
```diff
@@ -6,8 +6,10 @@ const appellantSubmissionRouter = require('./appellant-submission');
 const eligibilityRouter = require('./eligibility');
 const homeRouter = require('./home');
 const cookieRouter = require('./cookies');
+const guidancePagesRouter = require('./guidance-pages');
 
 router.use('/', homeRouter);
+router.use(guidancePagesRouter);
```
Comment (swdpcomputing) [inline packages/forms-web-app/src/routes/index.js:12]: Does this need an endpoint?

File: packages/forms-web-app/tests/unit/controllers/guidance-pages.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  let req;
+  let res;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+    jest.resetAllMocks();
+  });
+
+  describe('getBeforeAppeal', () => {
```
Comment (swdpcomputing) [inline packages/forms-web-app/tests/unit/controllers/guidance-pages.test.js:15]: This tests happy path, does it need testing to see if the correct error page displays if anything goes wrong?

File: e2e-tests/cypress/integration/common/guidance-pages.js (hunk 1 above)
Comment (davehlewis) [reply e2e-tests/cypress/integration/common/guidance-pages.js:5]: This is just doing the page navigation, plus the page navigation confirmation is in the Then step

File: e2e-tests/cypress/support/guidance-pages/guidancePageNavigation.js (hunk 2 above)
Comment (davehlewis) [reply e2e-tests/cypress/support/guidance-pages/guidancePageNavigation.js:11]: Good catch, fixing

File: packages/forms-web-app/src/routes/index.js (hunk 3 above)
Comment (davehlewis) [reply packages/forms-web-app/src/routes/index.js:12]: Nope, endpoints from the router and it's at base route

File: packages/forms-web-app/tests/unit/controllers/guidance-pages.test.js (hunk 4 above)
Comment (davehlewis) [reply packages/forms-web-app/tests/unit/controllers/guidance-pages.test.js:15]: We don't test this for anything else so would suggest not

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by swdpcomputing — APPROVED** [review]

**Pull request merged**